wagtail/core/blocks/list_block.py

```diff
@@ -98,6 +98,16 @@ def clean(self, value):
             else:
                 errors.append(None)
 
+        if self.meta.min_num is not None and self.meta.min_num > len(value):
+            non_block_errors.append(ValidationError(
+                _('The minimum number of items is %d') % self.meta.min_num
```

Comment:
Could we standardise on the brace format? I think we use this most often elsewhere and it might make the translators job a bit easier if they didn't need to know about all the older formats

Reply:
That's covered by #7111 - in the past the brace format caused problems because Transifex didn't validate that the placeholders matched in the translated string, leading to issues like https://github.com/wagtail/wagtail/issues/5539#issuecomment-527554273 - but that may have changed now. Either way, this is just copying the string that's already used in StreamBlock so it won't be adding any new work for translators.